src/System.Collections.Specialized/src/System/Collections/Specialized/OrderedDictionary.cs

```diff
@@ -395,7 +395,14 @@ public virtual void GetObjectData(SerializationInfo info, StreamingContext conte
 #endregion
 
 #region IDeserializationCallback implementation
-        void IDeserializationCallback.OnDeserialization(object sender)
+        void IDeserializationCallback.OnDeserialization(object sender) {
+            OnDeserialization(sender);
+        }
+        
+        /// <devdoc>
```

Comment:
Can you please exclude the "devdoc" comments as we already stripped them out in many places (including collections I believe), as they aren't used by anything and just add noise mostly.